src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -268,6 +268,9 @@ public async Task Put()
                         RecreateIndexes(databaseRecord);
                 }
 
+                if (databaseRecord.SupportedFeatures == null || databaseRecord.SupportedFeatures.Count == 0)
```

Comment:
since we have this in `CreateDatabase` this one is redundant, no?

Reply:
right, done :)